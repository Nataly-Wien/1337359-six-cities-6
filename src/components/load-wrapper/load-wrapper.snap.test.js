import React from 'react';
import {render} from '@testing-library/react';
import LoadWrapper from './load-wrapper';

it(`Should load screen render correctly`, () => {
  const isDataLoading = true;

  const {container} = render(<LoadWrapper isDataLoading={isDataLoading} />);

  expect(container).toMatchSnapshot();
});
