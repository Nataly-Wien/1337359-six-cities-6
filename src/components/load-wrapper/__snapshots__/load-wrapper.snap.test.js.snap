// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should load screen render correctly 1`] = `
<div>
  <div
    class="cities__status-wrapper"
    style="background-image: url(../img/spinner.gif); background-size: 90px 70px;"
  >
    <b
      class="cities__status"
    >
      Loading...
    </b>
  </div>
</div>
`;
